server/routes/api.js

```diff
@@ -0,0 +1,37 @@
+const express = require('express')
+const router = express.Router()
+
+const Transaction = require('../model/Transaction')
+
+
+router.get('/transactions', function (req, res) {
+    Transaction.find({}).exec(function (err, transactions) {
+        res.send(transactions)
+    })
+})
+
+router.post('/transaction', function(req, res){  
+    let newTransaction = req.body
+    let t1 = new Transaction(newTransaction)
+    t1.save(function(){
+        Transaction.find({}, function(err, transactions){
+            res.send(transactions)
+        })
+    })
+})
+
+router.delete('/transaction', function(req, res){
+    let id = req.body.id
+    Transaction.deleteOne({_id: id}, function(err, transaction){
+        res.send(`The Transaction ${id} was deleted successfully`)
```

Comment:
dont send back strings like that. Look into HTTP status codes to learn how to send resposnes